deps/chakrashim/core/lib/Runtime/Base/ThreadContext.h

```diff
@@ -309,13 +309,17 @@ class JITTimer
 class ThreadConfiguration
 {
 public:
-    ThreadConfiguration(bool enableExperimentalFeatures)
+    ThreadConfiguration(bool enableExperimentalFeatures, bool enableSimdjsFeature)
     {
         CopyGlobalFlags();
         if (enableExperimentalFeatures)
         {
             EnableExperimentalFeatures();
         }
+        if (!enableSimdjsFeature)
```

Comment:
Technically it should work, but there are many things that won't work if `enableExperimentalFeatures == false` :smile: